inc/msp430/msp430f5x_6x/Os_Internal_Arch_Cpu.h

```diff
@@ -0,0 +1,97 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice,
+ *    this list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+#ifndef _OS_INTERNAL_ARCH_CPU_H_
+#define _OS_INTERNAL_ARCH_CPU_H_
+
+/** \brief FreeOSEK Internal Architecture Cpu Dependent Header File
+ **
+ ** \file msp430/msp430f5xx_6xx/Os_Internal_Arch_Cpu.h
+ ** \arch msp430/msp430f5xx_6xx
+ **/
+
+/** \addtogroup FreeOSEK
+ ** @{ */
+/** \addtogroup FreeOSEK_Os
+ ** @{ */
+/** \addtogroup FreeOSEK_Os_Internal
+ ** @{ */
+
+/*****************************************************************************
+ * add your name to the developers and create for you a nick with
+ * 3 or 4 letters. Please do not use any given nick.
+ *
+ * PLEASE REMOVE THIS COMMENT
+ *****************************************************************************/
+
+/*****************************************************************************
+ * add a new version to this file, add the date, your initials and the main
+ * changes, only main changes shall be listed here. A detailed message log
+ * is saved in svn log and on the tracker system since every svn login message
+ * shalle indicate the related tracker id.
+ *
+ * PLEASE REMOVE THIS COMMENT
+ *****************************************************************************/
+/*
+ * modification history (new versions first)
+ * -----------------------------------------------------------
+ * 20140608 v0.1.1 PR   First version for LPC4337 microcontroller.
+ */
+
+/*==================[cpu macros]=============================================*/
+
+/* TODO: relocate these definitions */
+#define WORKING_FREQUENCY_MHZ       14.7456// 7.3728 //                         // in MHZ
```

Comment:
Pero es un float? Entonce va una F al final. Como querés que sea
interpretado el valor?
Am 06.07.2016 9:51 PM schrieb "fbucafusco" <notifications@github.com>:

> In inc/msp430/msp430f5x_6x/Os_Internal_Arch_Cpu.h
> <https://github.com/ciaa/firmware.modules.rtos/pull/32#discussion_r69799621>
> :
>
> > + * changes, only main changes shall be listed here. A detailed message log
> > + * is saved in svn log and on the tracker system since every svn login message
> > + * shalle indicate the related tracker id.
> > + *
> > + * PLEASE REMOVE THIS COMMENT
> > + *****************************************************************************/
> > +/*
> > + * modification history (new versions first)
> > + * -----------------------------------------------------------
> > + * 20140608 v0.1.1 PR   First version for LPC4337 microcontroller.
> > + */
> > +
> > +/*==================[cpu macros]=============================================*/
> > +
> > +/* TODO: relocate these definitions */
> > +#define WORKING_FREQUENCY_MHZ       14.7456// 7.3728 //                         // in MHZ
>
> Es una frecuencia que uso siempre. Podria subirla a 25. Esa frec hace que
> el baudrate de la uart calcule registros de forma exacta. Solo por eso..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/ciaa/firmware.modules.rtos/pull/32/files/6ef8a373a5ec7f8db6bb260a1f0a3ebeb8f29319#r69799621>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG7kmyaC1WjCd1u9WdFe-rDJMKQ6RtQpks5qTAcjgaJpZM4JFmjk>
> .
>